src/sass/components/_tour.scss

```diff
@@ -100,6 +100,17 @@
     fill: currentColor;
   }
 
+  &.is-viewed {
+    border: 1px solid $palette-green-90;
+    box-shadow: 0 0 2px $palette-green-70;
+    color: $palette-green-70;
+
+    &:hover,
+    &:focus {
+      color: $brand-background-dark;
+    }
```

Comment:
It looks to me like the same color is used for hover/focus normally (without is-viewed). I think that means we could move this setting in with the boxshadow on line 117? That's certainly not an important thing - but maybe a bit simpler, with less nesting?

Reply:
> 
> 
> It looks to me like the same color is used for hover/focus normally (without is-viewed). I think that means we could move this setting in with the boxshadow on line 117? That's certainly not an important thing - but maybe a bit simpler, with less nesting?

@mirisuzanne Totally overlooked that. Fixed!